packages/@aws-cdk/aws-applicationautoscaling/package.json

```diff
@@ -55,10 +55,13 @@
     "@aws-cdk/assert": "^0.12.0",
     "cdk-build-tools": "^0.12.0",
     "cfn2ts": "^0.12.0",
-    "pkglint": "^0.12.0"
+    "pkglint": "^0.12.0",
+    "fast-check": "^1.6.1"
```

Comment:
I like it when that remains sorted... But maybe that's just me :)

Reply:
No, you're right. And it has a pragmatic advantage too: slightly lower chance of conflicts when multiple people add packages.